dive_sailthru_client/client.py

```diff
@@ -129,8 +133,9 @@ def _infer_dive_publication(self, campaign):
             # blast list: "Utility Dive and sub pubs Blast List"
             # This regex handles that as well as normal blast lists
             return re.sub(r'( and sub pubs)? [Bb]last [Ll]ist$', '', list_name)
-        if dive_email_type in (DiveEmailTypes.HalfBlast, DiveEmailTypes.QuarterBlast, DiveEmailTypes.ThirdBlast) and "Group" in list_name:
-            return re.sub(r'( (Half|Thirds?|Quarters?))? Blast List - Group [A-D]$', '', list_name, flags=re.I)
+        if (dive_email_type in (DiveEmailTypes.HalfBlast, DiveEmailTypes.QuarterBlast, DiveEmailTypes.ThirdBlast, DiveEmailTypes.TwoThirdBlast) and
+            "Group" in list_name):
+            return re.sub(r'( (Half|(TWO )?Thirds?|Quarters?))? Blast List - Group [A-D]$', '', list_name, flags=re.I)
```

Comment:
Just for my own edification, is there some stakeholder-y reason for the all caps TWO?

Reply:
👌 + w/ the `re.I` flag it actually doesn't matter, just included because the rest was included in its expected case even tho it's case insensitive